integrationTests/chainSimulator/staking/delegation_test.go

```diff
@@ -852,3 +853,262 @@ func generateTransaction(sender []byte, nonce uint64, receiver []byte, value *bi
 		Signature: []byte(mockTxSignature),
 	}
 }
+
+// Test description
+//  Test that merging delegation  with whiteListForMerge and
+//  mergeValidatorToDelegationWithWhitelist contracts still works properly
+
+//  Test that their topups will merge too and will be used by auction list computing.
+//
```

Comment:
Wrong indentation + spacing + empty lines. Also, from what I remember, these tests should have the link number to the document

Reply:
updated comments